extensions/amp-story/0.1/related-articles.js

```diff
@@ -91,3 +91,25 @@ export function relatedArticlesFromJson(opt_articleSetsResponse) {
       return /** @type {!RelatedArticleSetDef} */ (articleSet);
     }));
 }
+
+/**
+ * @param {!JsonObject=} bookendComponents
+ * @return {!Array<!RelatedArticleSetDef>}
+ */
+export function parseArticlesToClassicApi(bookendComponents) {
+  const articleSet = {};
+  articleSet.articles = [];
+
+  bookendComponents.forEach(component => {
+    if (component.type != 'small') {
+      user().warn(component.type + ' is not supported in amp-story-0.1, ' +
```

Comment:
This is not necessarily true; they could put `component.type` as something completely unsupported.  I'm fine with hardcoding the list of stuff from 1.0 here.

Reply:
Done.